scripts/pause-sandbox

```diff
@@ -5,8 +5,8 @@
  */
 const fs = require('fs')
 const path = require('path')
-const osPath = require('ospath')
-const pauseFile = path.join(osPath.tmp(), '_pause-architect-sandbox-watcher')
+const os = require('os')
```

Comment:
Nice find, but this won't work, as Sandbox uses ospath. I'll switch this over to 'os' when I phase 'ospath' out of Sandbox.

Reply:
This is currently breaking on my machine for every script that calls this file, due to missing ospath on my machine.  Will ping on slack for the correct alternative.

Reply:
Undoing this, thanks.

Reply:
Ah, I hadn't run `npm install` lately, since the " Add Sandbox watcher pausing" commit which adds that package.  Thanks!

Reply:
Reverted, thank you.